modules/javafx.web/src/main/native/Source/ThirdParty/libxslt/src/libexslt/common.c

```diff
@@ -0,0 +1,124 @@
+#define IN_LIBEXSLT
```

Comment:
We don't include `libexslt` with jfxwebkit nor do we want to. Please revert the addition of the `libexslt/` directory tree.